### ISSUE TITLE
Build broken from pip with last commit

### ISSUE THREAD

**Issue by benburkhart1** [emdgroup/awscli-s3touch#1]
pip3 install https://github.com/merckgroup/awscli-s3touch/archive/master.zip

results in:

```
Collecting https://github.com/merckgroup/awscli-s3touch/archive/master.zip
  Downloading https://github.com/merckgroup/awscli-s3touch/archive/master.zip
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-k2hjnjzr-build/setup.py", line 5, in <module>
        long_description=open('README.rst').read(),
    FileNotFoundError: [Errno 2] No such file or directory: 'README.rst'
    
```

This appears to be due to this line:
https://github.com/emdgroup/awscli-s3touch/blob/master/setup.py#L5

caused by this commit which renamed it:
https://github.com/emdgroup/awscli-s3touch/commit/e3b1989afbf1fe52a94d885f25afc5c9cf4bf70b

**Comment by MxAR** [comment]
you can just rename the .zip archive before you install it with pip3, it works fine then ;)

**Comment by MxAR** [comment]
wait not the archive the README.md file to README.rst

**Comment by Nagarajuwas** [comment]
Hi I have rename the Readme file as. MD format then it is stsrted working. But when I run the s3 touch help in command prompt. It is saying the plugins is not found

**Comment by monken** [comment]
addressed with fef8f40edc457b945139b14bb2c88bde66b2f3d7
